server/libmain/libmain.go

```diff
@@ -218,14 +218,65 @@ func GetConfiguredEnvironmentOrDie(healthChecker *healthcheck.HealthChecker, app
 	return realEnv
 }
 
-func registerInternalGRPCServices(grpcServer *grpc.Server, env *real_environment.RealEnv) {
+func registerInternalGRPCServices(env *real_environment.RealEnv) error {
```

Comment:
I would call these methods something along the lines of `startInternalGRPCServers` rather than `register...` as you are not only registering gRPC services here but also starting the actual gRPC server.

Reply:
SG, done